src/views/catalog/CreateFromInstanceTypes/components/VMDetailsSection/components/VMDetailsBody/components/DetailsLeftGrid.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { Dispatch, SetStateAction, useMemo } from 'react';
+
+import { InstanceTypeState } from '@catalog/CreateFromInstanceTypes/utils/constants';
+import { V1beta1DataSource } from '@kubevirt-ui/kubevirt-api/containerized-data-importer/models';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { convertResourceArrayToMap } from '@kubevirt-utils/resources/shared';
+import { DescriptionList, GridItem, TextInput } from '@patternfly/react-core';
+import VirtualMachineDescriptionItem from '@virtualmachines/details/tabs/details/components/VirtualMachineDescriptionItem/VirtualMachineDescriptionItem';
+
+import useInstanceTypesAndPreferences from '../../../../../hooks/useInstanceTypesAndPreferences';
+import { InstanceTypeSizeDetails } from '../../../../SelectInstanceTypeSection/utils/types';
+import {
+  getCPUMemoryString,
+  getInstancetypeDetails,
+  getInstancetypeString,
+} from '../../../../SelectInstanceTypeSection/utils/utils';
+import { getOSFromDefaultPreference } from '../../../utils/utils';
+
+type DetailsLeftGridProps = {
+  vmName: string;
+  setVMName: Dispatch<SetStateAction<string>>;
+  bootSource: V1beta1DataSource;
+  instancetype: InstanceTypeState;
+};
+
+const DetailsLeftGrid: React.FC<DetailsLeftGridProps> = ({
+  bootSource,
+  instancetype,
+  vmName,
+  setVMName,
+}) => {
+  const { t } = useKubevirtTranslation();
+  const { preferences } = useInstanceTypesAndPreferences();
+  const preferencesMap = useMemo(() => convertResourceArrayToMap(preferences), [preferences]);
+  const operatingSystem = getOSFromDefaultPreference(bootSource, preferencesMap);
+
+  const { category, size } = instancetype;
+  const instancetypeSize: InstanceTypeSizeDetails = getInstancetypeDetails(category, size);
+
+  return (
+    <GridItem span={3}>
```

Comment:
Maybe the GridItem can be in the [VMDetailsBody.tsx](https://github.com/kubevirt-ui/kubevirt-plugin/pull/1027/files#diff-f32c1009121572a83f8303944d6e99532d7f84bf56071ad52cc3683113cc05e6) so if we want to change grid with something else, we have to change just one component

Reply:
Good idea! Done!